resources/turso/Cargo.toml

```diff
@@ -9,11 +9,11 @@ keywords = ["shuttle-service", "turso"]
 [dependencies]
 async-trait = "0.1.56"
 dunce = "1.0.4"
-libsql-client = "0.31.0" # Stays on 0.31 until https://github.com/libsql/libsql-client-rs/issues/52
+libsql = { version = "0.2.0", default-features = false, features = ["core", "remote"] }
```

Comment:
Is "core" needed for the local run with a local file database, or can we do without it?

Reply:
Yes, core is for local databases, remote for, well remote :smile: 
https://docs.rs/libsql/latest/libsql/struct.Database.html

Reply:
![image](https://github.com/shuttle-hq/shuttle/assets/56763273/ea34b44f-1dd4-4736-823c-92bb17779809)
I have only omitted the replication feature from the default features.